src/core/annotation.js

```diff
@@ -148,6 +112,57 @@ var Annotation = (function AnnotationClosure() {
   }
 
   Annotation.prototype = {
+    /**
+     * Set the border style (as AnnotationBorderStyle object).
+     *
+     * @public
+     * @memberof Annotation
+     * @param borderStyle {Dict} The border style dictionary
+     */
+    setBorderStyle: function Annotation_setBorderStyle(borderStyle) {
```

Comment:
It seems a bit weird to have a get and set that return/take different things. What about: 

1) In the constructor create the AnnotationBorderStyle and set it to borderStyle
2) have setBorderStyle become updateBorderStyle
3) remove the getters for border style and access the object directly